Aliases/map - move - over/5eb8d24b-d660-42e5-bbea-47a659bb0827 - Main GVAR.gvar

```diff
@@ -293,71 +322,120 @@ if c:
+   for addNewview in args.get('saveview'):
     if "," in addNewview:
      userPresets.update({addNewview.split(",")[0]:addNewview.split(",")[1]})
     else:
-     err("You must separate the key and value with a comma when adding `-newview` entries.")
+     err("You must separate the key and value with a comma when adding `-saveview` entries.")
    set_uvar("mapPresets",dump_json(userPresets))
    desc.append(f"Created the mapview preset `{addNewview.split(',')[0]}`")
-  #Here we can remove an already created preset
+   #Here we can remove an already created preset
   if args.last('deleteview'):
    for searchDeleteView in args.get('deleteview'):
     userPresets.pop(searchDeleteView)
    set_uvar("mapPresets",dump_json(userPresets))
    desc.append(f"Removed Preset `{searchDeleteView.upper()}`")
 </drac2>
-
+<drac2>
+# The default map bg's, created by @Nerds and Dragons#2817 
+presetMaps = load_json(get_gvar("a891401d-cdf2-44b2-b63b-bd86387c2659"))
+mapBG = presetMaps.copy()
+userMaps = load_json(get('mapBackgrounds','{}'))
+# Grab the user selected map, and update the main dict
+# This allows the user to replace/update items in the original dict
+mapBG.update(userMaps)
+# Iterate over the map dict, making it a little easier to read.
+# Only select the ones they are searching for, if they are searching
+maplist = [f"""**{maps}**""" for maps, over in mapBG.items() if args.last('search','').lower() in maps.lower()]
+# Split the list into groups of 20, to ensure it stays a reasonable size
+mapPagin = [maplist[i:i+20] for i in range(0, len(maplist), 20)]
+# Do we have a -loadmap?
+if args.last('mapload') or args.last('loadmap'):
+ # Loop through our users mapBackgrounds, looking for that map
+ argLoadmap = (args.last('loadmap')or args.last('mapload')).lower()
+ for maps in mapBG:
+  if argLoadmap in maps.lower():
+   foundMap = mapBG.get(maps)
+   if 'mapbg' in foundMap:
+    mapbg = foundMap.get('mapbg')
+   else:
+   	mapbg = ""
+   if 'mapsize' in foundMap:
+    mapsize = foundMap.get('mapsize')
+   if 'mapoptions' in foundMap:
+    mapoptions = foundMap.get('mapoptions')
+   else:
+   	mapoptions = ""
+   if 'walls' in foundMap:
+    walls = foundMap.get('walls')
+   else:
+   	walls = []
+   if 'objects' in foundMap:
+    objects = foundMap.get('objects')
+   else:
+   	objects = []
+   if typeof(foundMap) == 'SafeList':
+   	mapbg = foundMap[0]
+   	mapsize = foundMap[1]
+   	mapoptions = foundMap[2]
+   	objects = []
+   	walls = []
+   desc.append(f"Loaded the map {maps}")
+   neweffect = f"""{f"Size: {mapsize}" if mapsize else ""}{f" ~ Background: {mapbg}" if mapbg else ""}{f" ~ Options: {mapoptions}" if mapoptions else ""}{f' ~ Walls: {", ".join(walls)}' if walls else ""}{f' ~ Objects: {"/".join(objects)}' if objects else ""}{f" ~ View: {mapviewlocation}:{mapviewsize}" if mapviewsize and mapviewlocation else ""}""".strip(" ~")
+   if mapattach:
+    mapattach.add_effect('map', f"""-attack "||{neweffect}" """) 
+   break
+</drac2>
```

Comment:
New loadmap to work with the new savemap..it accounts for the older way the maps were being saved and will still load them